rsmtool/notebooks/comparison/score_distributions.ipynb

```diff
@@ -8,6 +8,41 @@
    ]
   },
   {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "old_consistency_file = join(output_dir_old, '{}_consistency.csv'.format(experiment_id_old))\n",
+    "new_consistency_file = join(output_dir_new, '{}_consistency.csv'.format(experiment_id_new))\n",
+    "\n",
+    "if exists(old_consistency_file) and exists(new_consistency_file):\n",
```

Comment:
And another minor consistency issue: we read all other outputs in the header and then pass them onto the other notebooks as a dictionary.